Core/NoOpRunner.Core/SpritesUriHandler.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace NoOpRunner.Core
+{
+    public static class SpritesUriHandler
+    {
+        private static string spritesDirectory = AppDomain.CurrentDomain.BaseDirectory + @"..\..\Sprites";
+        private static string spritesList = "SpritesList.txt";
+
+        private static Dictionary<string, Uri> sprites;
+
+        public static void LoadSprites()
+        {
+            sprites = new Dictionary<string, Uri>();
+
+            var spritesLines = File.ReadAllLines(spritesDirectory + "\\" + spritesList);
```

Comment:
Is there a reason why you need to load the sprite list dynamically? There is likely a nicer&better&proper way to do this using Resources.

Reply:
Now static, but it's hard(or i don't know how to use) to use and generate resources files using Rider

Reply:
@t-eliks Any insights / ideas?

Reply:
You should be able to add them as existing items to the solution and mark the build action as Embedded Resource. Then fetch them using Assembly.GetManifestResourceStream
https://docs.microsoft.com/en-us/dotnet/api/system.reflection.assembly.getmanifestresourcestream?view=netcore-3.1

Reply:
We will refactor this in a later PR